docs/user_guide/command_line_interface.rst

```diff
@@ -383,10 +383,11 @@ similar to the following.
     [info] [03/18/21 01:33:19.271] Num Eval Batches: 50
     [info] [03/18/21 01:33:19.271] Auc: 0.973, Avg Ranks: 24.477, MRR: 0.491, Hits@1: 0.357, Hits@5: 0.651, Hits@10: 0.733, Hits@20: 0.806, Hits@50: 0.895, Hits@100: 0.943
 
-After the training is finished, Marius will generate a directory according to the ``path.data_directory`` option for storing all the trained model
+After the training is finished, Marius will generate a directory according to the ``<path.data_directory>/<general.experiment_name>`` options for storing all the trained model
```

Comment:
Marius doesn't generate the directory after training. Better to word it something like:

"Marius stores the model and dataset in the ``<path.data_directory>/<general.experiment_name>/`` directory. If these options are not set in the configuration file, the default directory ``data/marius`` will be used".